commons/src/main/java/org/frankframework/util/CloseUtils.java

```diff
@@ -22,7 +22,8 @@
 
 @Log4j2
 public class CloseUtils {
-	private CloseUtils() {}
+	private CloseUtils() {
+	}
```

Comment:
Hier moet een comment in maar dat kunnen we niet 'enforcen' met een formatter, maar wel met `checkstyle`.